analysis/monkmistweaver/src/modules/shadowlands/covenant/FallenOrderCraneAverage.tsx

```diff
@@ -53,6 +53,9 @@ class FallenOrderCraneAverage extends Analyzer {
   }
 
   statistic() {
+    if (this.mwClones === 0 || this.soomCasts === 0 || this.mwClones === 0) {
```

Comment:
2 things
1) you are checking for mwClones === 0 twice for no reason

2) You shouldn't worry about soom or enm casts. If I'm reading this code correctly those being 0 won't make a crash and on top of that if someone summons 5 clones and get 0 enm and/or soom casts that would be a valid stat